lib/strip_js.ex

```diff
@@ -179,61 +183,59 @@ defmodule StripJs do
   possible for innocent CSS containing either of the strings `javascript:`
   or `expression(` to be mangled.
   """
-  @spec clean_css(String.t, opts) :: String.t
+  @spec clean_css(String.t(), opts) :: String.t()
   def clean_css(css, _opts \\ []) when is_binary(css) do
     css
     |> String.replace(~r/javascript \s* :/xi, "removed_by_strip_js:")
     |> String.replace(~r/expression \s* \(/xi, "removed_by_strip_js(")
   end
 
-
   ## Removes JS vectors from the given HTML attributes.
-  @spec clean_attrs([{String.t, String.t}]) :: [{String.t, String.t}]
+  @spec clean_attrs([{String.t(), String.t()}]) :: [{String.t(), String.t()}]
   defp clean_attrs(attrs) do
     attrs
     |> Enum.reduce([], &clean_attr/2)
-    |> Enum.reverse
+    |> Enum.reverse()
   end
 
   @attrs_with_urls ["href", "src", "background", "dynsrc", "lowsrc"]
 
-  @spec clean_attr({String.t, String.t}, [{String.t, String.t}]) :: [{String.t, String.t}]
+  @spec clean_attr({String.t(), String.t()}, [{String.t(), String.t()}]) :: [
+          {String.t(), String.t()}
+        ]
   defp clean_attr({attr, value}, acc) do
     attr = String.downcase(attr)
+
     cond do
-      (attr in @attrs_with_urls) && String.match?(value, ~r/^ \s* javascript \s* :/xi) ->
-        [{attr, "#"} | acc]  # retain the attribute so we emit valid HTML
+      attr in @attrs_with_urls &&
+          String.match?(value, ~r/^ \s* javascript \s* :/xi) ->
+        # retain the attribute so we emit valid HTML
+        [{attr, "#"} | acc]
+
       String.starts_with?(attr, "on") ->
-        acc  # remove on* handlers entirely
+        # remove on* handlers entirely
+        acc
+
       :else ->
-        [{attr, value |> escape_quotes |> html_escape} | acc]
+        [{attr, value} | acc]
     end
   end
 
-
   ## Performs good-enough HTML escaping to prevent HTML entity attacks.
-  @spec html_escape(String.t) :: String.t
+  @spec html_escape(String.t()) :: String.t()
   defp html_escape(html) do
     html
     |> String.replace("&", "&amp;")
```

Comment:
do we still need this

Reply:
we _do_ still need this however, because with this line:
https://github.com/appcues/strip_js/blob/nate/floki-updates-and-circle/lib/strip_js.ex

with `decode: false` Floki will actually only encode the HTML entities in data attributes, but not in the tree itself

Reply:
ah my bad didn't actually link to the line: https://github.com/appcues/strip_js/blob/nate/floki-updates-and-circle/lib/strip_js.ex#L240